lang/java/avro/pom.xml

```diff
@@ -164,6 +164,11 @@
       <artifactId>xz</artifactId>
       <version>${tukaani.version}</version>
     </dependency>
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <optional>true</optional>
```

Comment:
Can this be `provided` so it's not added to project classpaths for projects that depend on Avro.